docs/runtimes/notifications.md

```diff
@@ -0,0 +1,131 @@
+(notifications)=
+
+# Notifications
+
+MLRun supports configuring notifications on jobs and scheduled jobs. In this section, 
+the SDK for using notifications will be explained in detail.
+
+- [The Notification Object](#the-notification-object)
+- [Local vs Remote](#local-vs-remote)
+- [Notification Params and Secrets](#notification-params-and-secrets)
+- [Notification Kinds](#notification-kinds)
+- [Configuring Notifications For Runs](#configuring-notifications-for-runs)
+- [Configuring Notifications For Pipelines](#configuring-notifications-for-pipelines)
+- [Setting Notifications on Live Runs](#setting-notifications-on-live-runs)
+- [Setting Notifications on Scheduled Runs](#setting-notifications-on-scheduled-runs)
+- [Notification Conditions](#notification-conditions)
+
+
+## The Notification Object
+The notification object's schema is as follows:
+- `kind`: str - notification kind (slack, git, etc...)
+- `when`: list[str] - run states on which to send the notification (completed, error, aborted)
+- `name`: str - notification name
+- `message`: str - notification message
+- `severity`: str - notification severity (info, warning, error, debug)
+- `params`: dict - notification parameters (See definitions in [Notification Kinds](#notification-kinds))
+
+
+## Local vs Remote
+Notifications can either be sent locally from the SDK, or remotely from the MLRun API. 
```

Comment:
Notifications can be sent either locally from the SDK, or remotely from the MLRun API. Usually, a local run sends locally, and a remote run sends remotely.